mira/metamodel/ops.py

```diff
@@ -244,19 +251,26 @@ def stratify(
     for (source_stratum, target_stratum), concept in itt.product(structure, concept_map.values()):
         if concept.name in exclude_concepts:
             continue
-        param_name = f"p_{source_stratum}_{target_stratum}"
+        # Get stratum names from map if provided, otherwise use the stratum
+        src_strat_name = strata_name_map.get(source_stratum, source_stratum) \
+            if strata_name_map else source_stratum
+        tgt_strat_name = strata_name_map.get(target_stratum, target_stratum) \
```

Comment:
is there something gained from using cryptic variable names like `tgt` instead of `target`?

Reply:
Shorter lines ;) but yes, agreed, better variable names are needed

Reply:
Resolved in fae6a27599130d290cb86cd7e4d144265b42bac0